cmd/nri-kubernetes/main.go

```diff
@@ -67,7 +67,7 @@ func main() {
 		logger.SetLevel(log.DebugLevel)
 	}
 
-	i, err := createIntegrationWithHTTPSink(c.HTTPServerPort)
+	i, err := createIntegrationWithHTTPSink(c.HTTPServerPort, c.MaxRetries)
```

Comment:
Could we pass the whole config here? It will make easier to rebase #309

Reply:
Sure.